### PR TITLE
[TS-1289][TS-1255][TS-1286]: add test cases

### PR DESCRIPTION
**Pull request by xiaopingcs**
[TS-1289][TS-1255][TS-1286]: add test cases

### CODE REVIEW

File: tests/pytest/tools/taosdumpTest2.py [hunk 1]
```diff
@@ -91,6 +100,30 @@ def run(self):
         tdSql.query("select count(*) from t1")
         tdSql.checkData(0, 0, self.numberOfRecords)
 
+        tdSql.execute("create database test")
+        tdSql.execute("use test")
+        tdSql.execute("create table stb(ts timestamp, c1 binary(16000), c2 binary(16000), c3 binary(10000)) tags(t1 nchar(256))")
```
Comment (sangshuduo) [inline tests/pytest/tools/taosdumpTest2.py]: how about using a max boundary value to test the max length row?

File: tests/pytest/tools/taosdumpTest2.py [hunk 2]
```diff
@@ -91,6 +100,30 @@ def run(self):
...
+        tdSql.execute("drop database test")
+        tdSql.query("show databases")
+        tdSql.checkRows(0)
+
+        os.system("%staosdump -i /tmp" % binPath)
+
+        tdSql.execute("use test")
+        tdSql.error("show vnodes '' ")
+        tdSql.query("show stables")
+        tdSql.checkRows(1)
```
Comment (sangshuduo) [inline tests/pytest/tools/taosdumpTest2.py:124]: it would be better if we check the value with the old one to avoid it being trimmed.

**Review by sangshuduo — APPROVED** [review]
LGTM with suggestions.

File: tests/pytest/tools/taosdumpTest2.py (hunk 2 above)
Comment (xiaopingcs) [reply tests/pytest/tools/taosdumpTest2.py:124]: the value is also checked in line 122

File: tests/pytest/tools/taosdumpTest2.py (hunk 1 above)
Comment (xiaopingcs) [reply tests/pytest/tools/taosdumpTest2.py]: updated
